lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
```

Comment:
We will only use path on server side right ? when it  was zipped, all becomes folders already, why there is a problem when extracting on windows ?  

And for filename, both `\` and `/` should be removed. 

OK, I check the doc first, Thanks !